ls1intum/Artemis
# `Communication`: Update posts in the sidebar

### Checklist
#### General


#### Client


### Motivation and Context


Posts are not refreshed upon receiving a WebSocket message on the lecture and exercise details pages within the communication sidebar.

### Description

The criteria for refreshing posts has been extended to include the sidebar as well.

### Steps for Testing

Prerequisites:
- 2 students
- 1 Course with messaging and communication enabled

1. Write messages as student 1 on the messages page, on the communication page, on a lecture details page, and on an exercise details page.
2. Confirm with both student 1 and student 2, that new messages are visible without reloading the page
3. Disable communication and repeat step 1 and 2 (only messaging enabled)
4. Enable communication and disable messaging and repeat step 1 and 2 (only communication enabled)

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots


No UI changes
